docs/modules/nifi/pages/usage_guide/custom_processors.adoc

```diff
@@ -32,11 +33,13 @@ spec:
     custom: "docker.company.org/nifi:1.25.0-customprocessor"
 ----
 
-== Using the Official Image
-If you don't want to create a custom image, then you can use the extra volume mount functionality to load components and configure NiFi with to read these from the extra volumes.
+Also read the xref:guides:custom-images.adoc[Using customized product images] guide for additional information.
 
-For this to work you'll need to prepare a PVC containing your components.
-Usually the best way to do this will be to mount the PVC into a temporary container and then simply `kubectl cp` the nar files into that volume.
+== Using the official image
+If you don't want to create a custom image or don't have access to an image registry, you can use the extra volume mount functionality to mount a volume containing your custom components and configure NiFi to read these from the mounted volumes.
+
+For this to work you'll need to prepare a PersistentVolumeClaim (PVC) containing your components.
+Usually the best way to do this will be to mount the PVC into a temporary container and then `kubectl cp` the nar files into that volume.
```

Comment:
(part of the previous one)

```suggestion
Usually the best way to do this will be to mount the PVC into a temporary container and then `kubectl cp` the _nar_ files into that volume.
```